test/functionalTest/cases/1259_new_json_for_v2/putPostPatch_v2_entities_eid.test

```diff
@@ -0,0 +1,221 @@
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+PUT/POST/PATCH /v2/entities/eid tests
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v2/entities/E1 complete
+# 02. GET /v2/entities/E1
+#
+
+echo "01. POST /v2/entities/E1 complete"
+echo "================================="
+payload='{
+  "string": "a1",
+  "int":     19,
+  "float":   3.14,
+  "true":    true,
+  "false":   false,
+  "compound_vector":  [ 1,2,3 ],
+  "compound_object":  { "a": 1, "b": 2 },
+  "string_object": {
+    "value": "a1",
+    "type":  "TA1"
+  },
+  "int_object": {
+    "value": 21,
+    "type":  "TA1"
+  },
+  "float_object": {
+    "value": 22.1,
+    "type":  "TA1"
+  },
+  "true_object": {
+    "value": true,
+    "type":  "TA1"
+  },
+  "false_object": {
+    "value": false,
+    "type":  "TA1"
+  },
+  "object_object": {
+    "value": { "a": 1, "b": 2 },
+    "type":  "TA1"
+  },
+  "vector_object": {
+    "value": [ "a", "b" ],
+    "type":  "TA1"
+  },
+  "with_metadata01": {
+    "value": 0,
+    "type": "MT1",
+    "metadata": {
+      "md_int": 1,
+      "md_float:": 2.01,
+      "md_string": "STRING",
+      "md_true": true,
+      "md_false": false,
+      "md_object_int": {
+        "value": 1,
+        "type": "MDINT"
+      },
+      "md_object_float": {
+        "value": 2.01,
+        "type": "MDFLOAT"
+      },
+      "md_object_string": {
+        "value": "3",
+        "type": "MDSTRING"
+      },
+      "md_object_true": {
+        "value": true,
+        "type": "MDBOOL"
+      },
+      "md_object_false": {
+        "value": false,
+        "type": "MDBOOL"
+      }
+    }
+  }
+}'
+orionCurl --url /v2/entities/E1 --payload "$payload" --json
+echo
+echo
+
+
+echo "02. GET /v2/entities/E1"
+echo "======================="
+orionCurl --url /v2/entities/E1 --json
+echo
+echo
+
+
+
+--REGEXPECT--
+01. POST /v2/entities/E1 complete
+=================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E1
+Date: REGEX(.*)
+
+
+
+02. GET /v2/entities/E1
+=======================
+HTTP/1.1 200 OK
+Content-Length: 912
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "compound_object": {
```

Comment:
Renders has been adjusted to the new format in this PR?

(Just curiosity... it is ok to leave the step 2 output as it is now and adjust in a next PR).